### ISSUE TITLE
New Machine output setting only works when enabled as a global setting

### ISSUE THREAD

**Issue by Daeymon** [unexpectedpanda/retool#305]
The "Use `<machine>` instead of `<game>` in output DAT files" option does not work when enabled in "System settings". Only works when set in global settings.

Also, was there a reason to not have this setting be derivitive from the source file? For example
"Use `<machine>` instead of `<game>` in output DAT files if used in the original DAT."

An option like that, I could set globally, and all my CHD Redump and NoIntro dats would all process fine in one sweep.

**Comment by unexpectedpanda** [comment]
Because of the admittedly very unlikely but entirely possible scenario that someone could mix `<game>` and `<machine>` tags in the one DAT, and then I'd have to build a whole new system to follow that tag per-title.

As far as I'm aware ROM managers don't care which tag is used, and it doesn't change their behavior, so it's entirely an author preference thing. Have you seen otherwise?

**Comment by unexpectedpanda** [comment]
Okay, looks like I'll have to do a little testing there.

As for the system settings: I don't have access to my dev machine at the moment, but this was working when I implemented the feature. If I were to take a wild guess, when I hooked up the MAME Redump DAT files to the Redump clone lists afterwards, Retool decided the files were actually from Redump. This means they would share settings with the official Redump DAT files, opening up the opportunity for clashes. If this is the case, a bit more plumbing will have to be put in to still be able to use the Redump clone lists, but identify the MAME Redump DAT files as their own thing.
